pkg/standalone/standalone.go

```diff
@@ -1168,24 +1174,29 @@ func getPlacementImageName(imageInfo daprImageInfo, info initInfo) (string, erro
 		return "", err
 	}
 
-	image = getPlacementImageWithTag(image, info.runtimeVersion)
+	image = getPlacementImageWithTag(image, info.runtimeVersion, info.imageVariant)
 
 	// if default registry is GHCR and the image is not available in or cannot be pulled from GHCR
 	// fallback to using dockerhub.
 	if useGHCR(imageInfo, info.fromDir) && !tryPullImage(image) {
 		print.InfoStatusEvent(os.Stdout, "Placement image not found in Github container registry, pulling it from Docker Hub")
-		image = getPlacementImageWithTag(daprDockerImageName, info.runtimeVersion)
+		image = getPlacementImageWithTag(daprDockerImageName, info.runtimeVersion, info.imageVariant)
 	}
 	return image, nil
 }
 
-func getPlacementImageWithTag(name, version string) string {
-	if version == latestVersion {
-		return name
-	}
+func getPlacementImageWithTag(name, version, imageVariant string) string {
+	version = getVariantVersion(version, imageVariant)
 	return fmt.Sprintf("%s:%s", name, version)
 }
 
+func getVariantVersion(version string, imageVariant string) string {
```

Comment:
Can we get rid of duplication of code here and above in `kubernetes.go`?

Reply:
Done